rust-server/src/main.rs

```diff
@@ -468,8 +502,8 @@ fn sqlite_local_datetime_now_string() -> String {
     chrono::Local::now().to_rfc3339_opts(SecondsFormat::Millis, false)
 }
 
-enum Payload {
-    Binary,
-    Text,
-    Json,
-}
+// enum Payload {
```

Comment:
Yeah I should just burn this down. Nobody cares about unused code it's just wtf.